447 show buttons in workbatch according to configuration

GitEdvard opened the pull request:
Purpose:
Show buttons in a workbatch view according to a configuration provided by the api. 

Scope:
This is an implementation in the UI only. The corresponding endpoints /work-configurations and /events is not yet implemented in the backend. Furthermore, since a workbatch page is not yet in place, I started with one here and called it "example workbatch". This page only handles buttons, nothing else. 

Implementation:
I use the general actions creators in shared.js to create actions for the button clicked events as well as fetching a work configuration. 

Note:
I left some method name changes in workBatchDetials from previous changes. Most of them were changed back. Please have a look if these method names were the intended. They were inconsistent when I found them ("get" in the beginning or in the middle of the method name).

GitEdvard committed:
Add empty example workbatch page

with the intention to show buttons as defined in a configuration.

GitEdvard committed:
Send api request for button clicks

GitEdvard committed:
Render buttons on basis of workbatch configuration

Fix a consistent naming convention in workBatchDetails.

GitEdvard committed:
Fix action creator, get single entry

GitEdvard committed:
Change name work configuration --> work definition

mariya reviewed (changes requested):
The code looks fine, but I have some questions/comments about naming. Is the intention that ExampleWorkbatchContainer will at some point be edited to include the real implementation? If so, I think it is better to just name it WorkbatchContainer directly and just add TODO comments where applicable. If the plan is to replace ExampleWorkbatchContainer entirely in the future, then perhaps DummyWorkbatchContainer is a clearer name for something intended to be thrown away?

GitEdvard commented:
Yes, my intention was that ExampleWorkbatch would be used in future. So I'll rename it. Thanks for the comment.

GitEdvard committed:
Change name and split between container and presentation

GitEdvard commented:
I discovered that there are already an existing workbatch details container, and unfortunately this cannot be shown as the current state of the clims. Therefore I renamed my view to workBatchDetailsWaitingToBeMerged. (It will now show up next to the original view which is good). I also split up the code into a Container and Presentational class, in a similar manner as other views.

GitEdvard committed:
Change work batch endpoint name

mariya reviewed (approved):
This looks good to me, feel free to merge!
